horizons/world/building/building.py

```diff
@@ -301,7 +260,7 @@ def getInstance(cls, session, x, y, action='idle', level=0, rotation=45, action_
 				# set first action
 				action = list(action_set.keys())[0]
 
-		instance.actRepeat(action + "_" + str(action_set_id), facing_loc)
+		instance.actRepeat(action + "_" + str(action_set_id), rotation)
```

Comment:
What about using `str.format()` instead?
```python
instance.actRepeat("{0}_{1:d}".format(action, action_set_id), rotation)
```

Reply:
sure

Reply:
Is this ok too?
    instance.actRepeat("{}_{}".format(action, action_set_id), rotation)

Reply:
Yes, but for extra safety checks the type of the format field should also be specified. That way Python will throw an exception if the type is violated and the error will not propagate further.

Reply:
Appearently action_set_id is actually always a string.
It's something like "as_lumberjack_barrack0".